app/assets/javascripts/discourse/app/controllers/user-topics-list.js

```diff
@@ -36,9 +28,9 @@ export default Controller.extend(BulkTopicSelection, {
     this.set("application.showFooter", !this.get("model.canLoadMore"));
   },
 
-  @discourseComputed("incomingCount")
-  hasIncoming(incomingCount) {
-    return incomingCount > 0;
+  @discourseComputed("pmTopicTrackingState.newIncoming.[]")
```

Comment:
Didnt try but would this just work ?

```
incomingCount: computed("pmTopicTrackingState.newIncoming.length")
```

Reply:
I changed it to `incomingCount: alias("pmTopicTrackingState.newIncoming.length"),`